Lib/test/test_cmath.py

```diff
@@ -55,10 +55,7 @@ class CMathTests(unittest.TestCase):
     # commented out to allow incremented addition of functions.
     #
     # list of all functions in cmath
-    # test_functions = [getattr(cmath, fname) for fname in [
-    #        'acos', 'acosh', 'asin', 'asinh', 'atan', 'atanh',
-    #        'cos', 'cosh', 'exp', 'log', 'log10', 'sin', 'sinh',
-    #        'sqrt', 'tan', 'tanh']]
+    test_functions = [getattr(cmath, fname) for fname in ['sqrt']]
```

Comment:
I was hoping let's keep the above commented lines intact in the comment itself and create a new variable.
Since I'm also working on this, we can easily compare what all is supposed to be implemented and what is already implemented.

Reply:
> we can easily compare what all is supposed to be implemented and what is already implemented

That's basically what the issue is there for. If something isn't implemented, you'd see it in the issue (instead of browsing to a test file).